source/frontend/src/hooks/useProjectTypeahead.ts

```diff
@@ -27,7 +27,8 @@ export const useProjectTypeahead = (): IProjectTypeahead => {
       if (isMounted()) {
         setMatchedProjects(
           data.map<IAutocompletePrediction>(x => {
-            return { id: x.id || 0, text: x.description?.toString() || '' };
+            let projectText = `${x.code || ''}  ${x.description}`;
```

Comment:
umm, do we need || '' for description?

Reply:
probably is not needed since description is never null. Will add it for the interface definition.